### PR TITLE
Claims v1 copy take 2

### PR DESCRIPTION
**Pull request by palmfjord**

### CODE REVIEW

File: src/main/java/com/hedvig/botService/chat/ClaimsConversation.java [hunk 1]
```diff
@@ -103,72 +104,39 @@
...
-    createMessage(MESSAGE_CLAIMS_ASK_PHONE_END, new MessageBodyParagraph("Toppen!"), 2000);
-
-    addRelay(MESSAGE_CLAIMS_ASK_PHONE_END, MESSAGE_CLAIMS_RECORD);
+    createMessage(MESSAGE_CLAIMS_ASK_PHONE, new MessageBodyNumber("Om jag skulle behöva kontakta dig över telefon efteråt, vad är ditt nummer?"));
 
     createMessage(
-      MESSAGE_CLAIMS_RECORD,
-      new MessageBodyParagraph("Berätta vad som har hänt genom att spela in ett röstmeddelande"),
+      MESSAGE_CLAIMS_RECORD_1,
+      new MessageBodyParagraph("Perfekt! Nu ska du få göra röstinspelningen. Försök besvara de här frågorna så utförligt du kan:"),
```
Comment (meletios) [inline src/main/java/com/hedvig/botService/chat/ClaimsConversation.java]: do we need the last colon?

File: src/main/java/com/hedvig/botService/chat/ClaimsConversation.java [hunk 2]
```diff
@@ -343,9 +305,10 @@ public void receiveEvent(EventTypes e, String value, UserContext userContext) {
         String relay = getRelay(value);
         if (relay != null) {
           completeRequest(relay, userContext);
+          break;
         }
 
-        if (value.equals(MESSAGE_CLAIMS_OK)) {
+        if (value.equals("message.claims.chat")) {
```
Comment (meletios) [inline src/main/java/com/hedvig/botService/chat/ClaimsConversation.java]: Shall we use a const here?

File: src/main/java/com/hedvig/botService/chat/ClaimsConversation.java (hunk 1 above)
Comment (palmfjord) [reply src/main/java/com/hedvig/botService/chat/ClaimsConversation.java]: it's in the design so i assumed yes

File: src/main/java/com/hedvig/botService/chat/ClaimsConversation.java (hunk 2 above)
Comment (palmfjord) [reply src/main/java/com/hedvig/botService/chat/ClaimsConversation.java]: yes

**Pull request merged**